tests/core/starlark/common_tests.bzl

```diff
@@ -0,0 +1,32 @@
+load("@io_bazel_rules_go//go/private:common.bzl", "has_shared_lib_extension")
+load("@io_bazel_rules_go//go/private:skylib/lib/unittest.bzl", "asserts", "unittest")
+
+def _versioned_shared_libraries_test(ctx):
```

Comment:
I'm glad to have a unit test for this, but could you also add something in `tests/core/cgo` that confirms a library with a name like this can be linked?

Please make sure to update README.rst (and add one in this directory) and run `tests/update.py`, too. Test build files aren't readable on their own, so we keep test docs separate.

Reply:
Created (and updated) the tests, thanks!